rfcs/formats/network/iin_method.md

```diff
@@ -0,0 +1,264 @@
+
+> Note: During NetworkDID creation, the `updatePolicy` of the `GroupMultiSig` verificationMethod is not used. Instead. the request has to be authenticated against all the participants in `networkParticipants` list.
+
+Example NetworkDID creation request:
+
+```json
+{
+  "NetworkDIDDocument": {
+    "id": "did:iin:<iin_name>:<network_name>",
+    "networkParticipants": [
+      "did:iin:<iin_name>:<network_participant_1>",
+      "did:iin:<iin_name>:<network_participant_2>",
+      "did:iin:<iin_name>:<network_participant_3>"
+    ],
+    "verificationMethod": [{
+        "id": "did:iin:<iin_name>:<network_name>#multisig",
+        "type": "GroupMultiSig",
+        "controller": "did:iin:<iin_name>:<network_name>",
+        "multisigKeys": [
+          "did:iin:<iin_name>:<network_participant_1>#key1",
+          "did:iin:<iin_name>:<network_participant_2>#key3",
+          "did:iin:<iin_name>:<network_participant_3>#key1"
+        ],
+        "updatePolicy": {
+          "id": "did:iin:<iin_name>:<network_name>#updatepolicy",
+          "controller": "did:iin:<iin_name>:<network_name>",
+          "type": "VerifiableCondition2021",
+          "conditionAnd": [{
+              "id": "did:iin:<iin_name>:<network_name>#updatepolicy-1",
+              "controller": "did:iin:<iin_name>:<network_name>",
+              "type": "VerifiableCondition2021",
+              "conditionOr": ["did:iin:<iin_name>:<network_participant_3>#key1",
+                "did:iin:<iin_name>:<network_participant_2>#key3"
+              ]
+            },
+            "did:iin:<iin_name>:<network_participant_1>#key1"
+          ]
+        }
+      },
+
+      {
+        "id": "did:iin:<iin_name>:<network_name>#fabriccerts",
+        "type": "DataplaneCredentials",
+        "controller": "did:iin:<iin_name>:<network_name>",
+        "FabricCredentials": {
+          "did:iin:<iin_name>:<network_participant_1>": "Certificate3_Hash",
+          "did:iin:<iin_name>:<network_participant_2>": "Certificate2_Hash",
+          "did:iin:<iin_name>:<network_participant_3>": "Certificate3_Hash"
+        }
+      }
+    ],
+    "authentication": [
+      "did:iin:<iin_name>:<network_name>#multisig"
+    ],
+    "relayEndpoints": [{
+        "hostname": "10.0.0.8",
+        "port": "8888"
+      },
+      {
+        "hostname": "10.0.0.9",
+        "port": "8888"
+      }
+
+    ]
+  },
+  "signatures": {
+    "did:iin:<iin_name>:<network_participant_1>": "...",
+    "did:iin:<iin_name>:<network_participant_2>": "...",
+    "did:iin:<iin_name>:<network_participant_3>": "..."
+  }
+}
+
+```
+
+How this request is created is out of the scopt of this specification. However, generally it is recommended that the signatures of the request are collected through a smart contract in the network blockchain itself.
```

Comment:
"out of the scopt" --> "beyond the scope"

Reply:
Since this document is only about the DID method, I am not including the flow here. Flow is appropriate only if a smart contract based request generation is not possible. In fabric it can be done through smart contracts only.

Reply:
I was commenting on the accuracy of saying "collected through a smart contract". The process of collection is above the smart contract, which validates the collection result, right?